pkg/config/env.go

```diff
@@ -212,4 +212,22 @@ var Config = struct {
 	// UI path  => localhost:18000/foo"
 	// API path => localhost:18000/foo/api/v1"
 	WebPrefix string `env:"FLAGR_WEB_PREFIX" envDefault:""`
+
+	// rules for validating flags on enable - when a flag is updated or enabled
```

Comment:
I think `Enabled...` is not special here. Maybe we can just call it `CustomFlagValidationRules`, and uses a hook pattern to call this `CustomFlagValidationRules` whenever there's a write path change to the specified flag.

Reply:
I think I agree.  What we specifically want to allow is users to create a fresh flag (especially since the UI only supports a default / 'simple boolean' new flag template (https://github.com/checkr/flagr/issues/428)), then tweak it until it's ready to turn on, and only when it's turned on require the additional validation step.  I think that's consistent with renaming things as you suggested.